app/controllers/api/databases.php

```diff
@@ -2790,3 +2873,98 @@ function createAttribute(string $databaseId, string $collectionId, Document $att
 
         $response->dynamic($usage, Response::MODEL_USAGE_COLLECTION);
     });
+
+App::get('/v1/databases/:databaseId/slow-queries')
+    ->desc('List Slow Queries Documents')
+    ->groups(['api', 'database'])
+    ->label('docs', false)
+    ->label('scope', 'documents.read')
+    ->label('usage.metric', 'documents.{scope}.requests.read')
+    //->label('usage.params', ['databaseId:{request.databaseId}', 'collectionId:{request.collectionId}'])
+    ->label('sdk.auth', [APP_AUTH_TYPE_ADMIN])
+    ->label('sdk.namespace', 'databases')
+    ->label('sdk.method', 'listSlowQueries')
+    ->label('sdk.description', '/docs/references/databases/list-slow-queries-documents.md')
```

Comment:
Let's rename this to be `list-slow-queries` for consistency

Reply:
done!